plutus-core/plutus-core/src/PlutusCore/Evaluation/Machine/ExBudget.hs

```diff
@@ -146,26 +145,12 @@ where
 
 import           PlutusPrelude                          hiding (toList)
 
-import           PlutusCore.Core
-import           PlutusCore.Name
-
 import           Data.Semigroup
 import           Data.Text.Prettyprint.Doc
 import           Deriving.Aeson
 import           Language.Haskell.TH.Lift               (Lift)
 import           PlutusCore.Evaluation.Machine.ExMemory
 
-class ToExMemory term where
-    -- | Get the 'ExMemory' of a @term@. If the @term@ is not annotated with 'ExMemory', then
-    -- return something arbitrary just to fit such a term into the builtin application machinery.
-    toExMemory :: term -> ExMemory
```

Comment:
Now that we don't have a separate builtin application machinery, we can just define `toExMemory` locally in each evaluator that needs it (i.e. only the CEK machine) and avoid the type class dispatching.

Reply:
nice